test/common/constants.t.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+address constant CONTRACT_DEPLOYER = 0x76bd253e7a0FB5896b4ACA4b9ef06E9ee2b74e8E;
+
+address constant TEST_ERC20_TOKEN = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
```

Comment:
It would be helpful to add a comment specifying that this token is USDC on Ethereum